src/AnnotationThread.js

```diff
@@ -199,8 +200,10 @@ class AnnotationThread extends EventEmitter {
                 isPending={isPending}
                 headerHeight={this.headerHeight}
             />,
-            util.getPopoverLayer(pageEl)
+            popoverLayer
         );
+
+        this.scrollIntoView();
```

Comment:
Is this necessary? If the `position()` call already positions the annotation popover within the viewport, do we need to center it as well?

Reply:
Nope. It's actually only needed for when we want to scroll to an annotation from the sidebar (in the future). So removing this for now